acceptance/tests/ticket_1429_facter_with_strict_flag.rb

```diff
@@ -0,0 +1,11 @@
+test_name 'ticket FACT-1429  facter should return exit code 1 for querying non-existing fact when --strict is on'
```

Comment:
we should create a testrail case for this and place its number (including the 'C') where `ticket FACT-1429` is above.

when the case is creaeted, please reference that ticket, and create a new ticket in the same sprint with a link to it from the test case in the automation ticket field.  also place a reference to the automation ticket in the reference field (with commas separating the references).

then you can set FACT-1429 `qa-status` to `reviewed`